src/odemis/driver/technolution.py

```diff
@@ -1131,13 +1197,13 @@ def _setFilename(self, file_name):
 
     def _setCellTranslation(self, cellTranslation):
         """
-        Setter for the cell translation, each cell has a translation (overscan_parameters) stored as an "x,y" (tuple)
-        which packed with all the translations of a full row in a tuple. Which in its turn is packed in a
-        tuple  with all the columns together.
+        Setter for the cell translation, each cell has a translation (overscan_parameters) stored as an "x,
+        y" (tuple of ints) which is packed with all the translations of a full row in a tuple. Which in its turn is
+        packed in a tuple  with all the columns together.
```

Comment:
Each x, y cell translation (overscan parameters) is stored as a tuple of two ints. The cell translations of all cells for a row are nested in another tuple. And finally, all cell translations per row are nested in another tuple representing the columns. (((int, int), (int, int)....), ((int, int), (int, int), .....), ......).
Like this?

Reply:
rephrase and check what it is and add the array as above.